src/js/Alert.js

```diff
@@ -27,7 +27,7 @@ class Alert extends React.Component {
     }
 
     pressResetTimeoutButton(event) {
-        let count = store.getState().ui[store.getState().activeApp].resetTimeout.resetTimeoutValue / 1000;
+        let count = store.getState().ui[store.getState().activeApp].resetTimeoutValue / 1000;
```

Comment:
@LiliiaShlikhtLuxoft As you can see, seems the `resetTimeoutValue` is relates to the application object.

Reply:
@ValeriiMalkov fixed in [commit](https://github.com/LuxoftSDL/generic_hmi/pull/22/commits/8c894f8acca6943cf8ee919149e9ed969815c680)

Reply:
@ValeriiMalkov please check fix [commit](https://github.com/LuxoftSDL/generic_hmi/pull/22/commits/b9fd58610789f82c517b149bbf277190da49c463)